### PR TITLE
Move feature toggles out of the dungeon to commons

### PR DESCRIPTION
**Pull request by MishaDemianenko**
Feature toggles are commonly used across the modules and there is no
reason to keep them in a package that is not related to what they are
doing.

### CODE REVIEW

File: community/common/src/main/java/org/neo4j/helpers/FeatureToggles.java [hunk 1]
```diff
@@ -17,7 +17,7 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-package org.neo4j.unsafe.impl.internal.dragons;
+package org.neo4j.helpers;
```
Comment (chrisvest) [inline community/common/src/main/java/org/neo4j/helpers/FeatureToggles.java]: Does this mean it's public API now? I'd prefer to keep it non-public.

**Review by chrisvest — COMMENTED** [review]
Just one comment.

File: community/common/src/main/java/org/neo4j/helpers/FeatureToggles.java (hunk 1 above)
Comment (MishaDemianenko) [reply community/common/src/main/java/org/neo4j/helpers/FeatureToggles.java]: Its really good point; thats wrong them - have moved into another package `org.neo4j.util` inside commons

**Comment by MishaDemianenko** [comment]
@chrisvest updated

**Review by chrisvest — APPROVED** [review]

**Pull request merged**